src/axelar/AxelarTranceiver.sol

```diff
@@ -0,0 +1,130 @@
+
+    error UnsupportedMessageType();
+    error InvalidSibling(uint16 chainId, string sourceChain, string sourceAddress);
+    error NotImplemented();
+
+    constructor(address _gateway, address _gasService, address _manager)
+        AxelarExecutable(_gateway)
+        Managed(_manager)
+    {
+        gasService = IAxelarGasService(_gasService);
+    }
+
+    function _setup(bytes calldata params) internal override {
+
+    }
+
+    function contractId() external pure override returns (bytes32) {
+        return keccak256('axelar-transceiver');
+    }
+
+    /**
+     * Set the bridge manager contract address
+     * @param chainId The chainId of the chain. This is used to identify the chain in the EndpointManager.
+     * @param chainName The chainName of the chain. This is used to identify the chain in the AxelarGateway.
+     */
+    function setAxelarChainId(uint16 chainId, string calldata chainName, string calldata axelarAddress) external onlyOwner {
+        idToAxelarChainIds[chainId] = chainName;
+        axelarChainIdToId[chainName] = chainId;
+        idToAxelarAddress[chainId] = axelarAddress;
+        axelarAddressToId[axelarAddress] = chainId;
+    }
+
+    /// @notice Fetch the delivery price for a given recipient chain transfer.
+    /// param recipientChain The Wormhole chain ID of the target chain.
+    /// param instruction An additional Instruction provided by the Transceiver to be
+    ///        executed on the recipient chain.
+    /// @return deliveryPrice The cost of delivering a message to the recipient chain,
+    ///         in this chain's native token.
+    function quoteDeliveryPrice(
+        uint16 /*recipientChain*/,
+        TransceiverStructs.TransceiverInstruction memory /*instruction*/
+    ) external view override virtual returns (uint256) {
+        // Axelar doesn't support on-chain gas fee.
+        return 0;
+    }
+
+    /// @dev Send a message to another chain.
+    /// @param recipientChainId The Wormhole chain ID of the recipient.
+    /// param instruction An additional Instruction provided by the Transceiver to be
+    /// executed on the recipient chain.
+    /// @param nttManagerMessage A message to be sent to the nttManager on the recipient chain.
+    function sendMessage(
+        uint16 recipientChainId,
+        TransceiverStructs.TransceiverInstruction memory /*instruction*/,
+        bytes memory nttManagerMessage,
+        bytes32 recipientNttManagerAddress,
+        bytes32 refundAddress
+    ) external payable override virtual onlyManager() {
+        emit SendTransceiverMessage(recipientChainId, nttManagerMessage, recipientNttManagerAddress, refundAddress);
+
+        string memory destinationContract = idToAxelarAddress[recipientChainId];
+        string memory destinationChain = idToAxelarChainIds[recipientChainId];
+
+        if(bytes(destinationChain).length == 0 || bytes(destinationContract).length == 0) revert InvalidChainId(recipientChainId);
```

Comment:
recommend including the destination chain string in the error message

Reply:
did it this way to match your errors, i can change if you want.

Reply:
yes - im suggesting including this since this maps `recipientChainIds` to `destinationChain` - likely useful for the client to check if there's an error.